YoutubeExplode/Playlists/PlaylistController.cs

```diff
@@ -47,16 +47,65 @@ public PlaylistController(HttpClient http)
         var raw = await SendHttpRequestAsync(request, cancellationToken);
         var playlist = PlaylistExtractor.Create(raw);
 
-        if (!playlist.IsPlaylistAvailable())
+        if (!playlist.IsPlaylistDetailsAvailable())
+        {
+            //try getting info from the next endpoint if this one can't get the playlist
+            playlist = await GetPlaylistVideosAsync(playlistId,cancellationToken);
+        }
```

Comment:
Does this ever happen?

Reply:
yes, the playlist id that is used in #555 or this playlists id ```RDMM3Kuw0ZwI0WY``` would get the metadata from next endpoint.